app/src/main/java/com/clearkeep/screen/auth/login/LoginViewModel.kt

```diff
@@ -27,6 +38,40 @@ class LoginViewModel @Inject constructor(
     val passError: LiveData<String>
         get() = _passError
 
+    val googleSignIn: GoogleSignInOptions =
+        GoogleSignInOptions
+            .Builder(GoogleSignInOptions.DEFAULT_SIGN_IN)
+            .requestIdToken("1092685949059-31fdlstnskjc799is97moat5bagskrsr.apps.googleusercontent.com")
+            .requestProfile()
```

Comment:
.requestIdToken(getString(R.string.default_web_client_id))

fix token in string xml
AND maybe move this variable "googleSignIn" to LoginActivity directly